appcues/src/main/java/com/appcues/trait/appcues/TooltipsPaths.kt

```diff
@@ -74,18 +76,50 @@ internal sealed class TooltipPointerPosition {
 
     object Bottom : TooltipPointerPosition() {
 
+        override val isVertical = true
+
         override fun toContainerCornerRadius(cornerRadius: Dp): ContainerCornerRadius {
             return ContainerCornerRadius(
                 topStart = cornerRadius,
                 topEnd = cornerRadius,
-                bottomStart = if (alignment.value.horizontalAlignment == START) 0.dp else cornerRadius,
-                bottomEnd = if (alignment.value.horizontalAlignment == END) 0.dp else cornerRadius
+                bottomStart = if (alignment.value.horizontalAlignment == PointerHorizontalAlignment.LEFT) 0.dp else cornerRadius,
+                bottomEnd = if (alignment.value.horizontalAlignment == PointerHorizontalAlignment.RIGHT) 0.dp else cornerRadius
+            )
+        }
+    }
+
+    object Left : TooltipPointerPosition() {
+
+        override val isVertical = false
+
+        override fun toContainerCornerRadius(cornerRadius: Dp): ContainerCornerRadius {
+            return ContainerCornerRadius(
+                topStart = if (alignment.value.verticalAlignment == PointerVerticalAlignment.TOP) 0.dp else cornerRadius,
+                topEnd = cornerRadius,
+                bottomEnd = cornerRadius,
+                bottomStart = if (alignment.value.verticalAlignment == PointerVerticalAlignment.BOTTOM) 0.dp else cornerRadius
+            )
+        }
+    }
+
+    object Right : TooltipPointerPosition() {
+
+        override val isVertical = false
+
+        override fun toContainerCornerRadius(cornerRadius: Dp): ContainerCornerRadius {
+            return ContainerCornerRadius(
+                topStart = cornerRadius,
+                topEnd = if (alignment.value.verticalAlignment == PointerVerticalAlignment.TOP) 0.dp else cornerRadius,
+                bottomEnd = if (alignment.value.verticalAlignment == PointerVerticalAlignment.BOTTOM) 0.dp else cornerRadius,
+                bottomStart = cornerRadius,
             )
         }
     }
 
     object None : TooltipPointerPosition() {
 
+        override val isVertical = false
```

Comment:
Does this case represent when no position preference is given in the builder?

Reply:
ah right, yeah i think the floating bottom case is valid, might be useful for toast like things. I recall something similar in a competitor product as well.